manifests/static.pp

```diff
@@ -0,0 +1,16 @@
+# @summary Manage the static files (assets)
+# @api private
+class pulpcore::static {
+  file { $pulpcore::static_root:
```

Comment:
I think this parameter `$pulpcore::static_root` was present in your other PR but didn't get included here, which is breaking things

Otherwise it looks good to me

Reply:
Ah yes, here it was still `pulp_static_root`.